db/seeds/protocols/student.rb

```diff
@@ -55,6 +55,7 @@
 nm_measurement = protocol.measurements.find_by_questionnaire_id(nameting_id)
 nm_measurement ||= protocol.measurements.build(questionnaire_id: nameting_id)
 nm_measurement.open_from_offset = 30.weeks + 3.days + 13.hours # Thursday 1pm last week
+nm_measurement.stop_measurement = true
```

Comment:
Eigenlijk moeten we van alle andere vragenlijst-seeds deze waarde expliciet op false zetten. Dat is de default, maar die zou dus in theorie overschreven kunnen worden en de seeds zorgen er dan ook voor dat we alles gelijktrekken.